charts/vela-core/templates/kubevela-controller.yaml

```diff
@@ -112,6 +112,9 @@ spec:
             - "--webhook-port={{ .Values.webhookService.port }}"
             - "--webhook-cert-dir={{ .Values.admissionWebhooks.certificate.mountPath }}"
             {{ end }}
+            {{ if not .Values.useAppConfig }}
+            - "--app-config-installed=false"
+            {{ end }}
```

Comment:
can we add a conditional statement for the crd ?

Reply:
is there a way to not install the appConfig CRD?

Reply:
I'm not sure, maybe we need create two charts

Reply:
One is application only(name is `kubevela`), one is application with AppConfig(name is `kubevela-compatible`).